diff --git a/Modules/ThirdParty/TIFF/src/itktiff/tif_config.h.in b/Modules/ThirdParty/TIFF/src/itktiff/tif_config.h.in
index ce70dc7626..19d30b5b2e 100644
--- a/Modules/ThirdParty/TIFF/src/itktiff/tif_config.h.in
+++ b/Modules/ThirdParty/TIFF/src/itktiff/tif_config.h.in
@@ -420,7 +420,7 @@ the sizes can be different.*/
 /* MSVC does not support C99 inline, so just make the inline keyword
    disappear for C.  */
 #ifndef __cplusplus
-#  ifdef _MSC_VER
+#  if defined(_MSC_VER) && _MSC_VER < 1900
 #    define inline
 #  endif
 #endif
